app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -174,7 +174,7 @@ def check_policy_prevent(policy_event, event_target, cb_method)
       :class_name  => self.class.to_s,
       :instance_id => id,
       :method_name => :check_policy_prevent_callback,
-      :args        => [cb_method, event_target],
+      :args        => [cb_method, event_target.class.name, event_target.id],
```

Comment:
@kbrock actually that wasn't the correct answer, 
This is where the parameters for the callback are built:
https://github.com/ManageIQ/manageiq/blob/b3574a3b4414818026b3b26e715fb76b95e72fba/app/models/miq_queue.rb#L412

you wrote that line actually :bowtie: